docs/tutorial/volumes/policy.md

```diff
@@ -0,0 +1,307 @@
+        .
+        .
+        .
+```
+### Replica Affinity:
+
+For StatefulSet applications, to distribute single replica volume on specific cstor pool we can use the `replicaAffinity` enabled scheduling.
+This feature should be used with delay volume binding i.e. `volumeBindingMode: WaitForFirstConsumer` in StorageClass as shown below.
+
+If `WaitForFirstConsumer` volumeBindingMode is set, then the csi-provisioner will wait for the scheduler to pick a node. The topology of
+that selected node will then be set as the first entry in preferred list and will be used  by the volume controller to create the volume
+replica on the cstor pool scheduled on preferred Node. 
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: cstor-sparse-auto
+provisioner: cstor.csi.openebs.io
+allowVolumeExpansion: true
+volumeBindingMode: WaitForFirstConsumer
+parameters:
+  replicaCount: "1"
+  cstorPoolCluster: "cspc-disk"
+  cas-type: "cstor"
+  cstorVolumePolicy: "csi-volume-policy"      // policy created with replicaAffinity set to true
+```
+
+This requires to be enabled via volume policy before provisioning the volume
+
+```yaml
+apiVersion: cstor.openebs.io/v1
+kind: CStorVolumePolicy
+metadata:
+  name: csi-volume-policy
+  namespace: openebs
+spec:
+  provision:
+    replicaAffinity: true
+```
+
+### Volume Target Pod Affinity:
+
+The Stateful workloads access the OpenEBS storage volume by connecting to the Volume Target Pod. 
+Target Pod Affinity policy can be used to co-locate volume target pod on the same node as workload.
+This feature makes use of the Kubernetes Pod Affinity feature that is dependent on the Pod labels. 
+User will need to add the following label to both Application and volume Policy.
+
+Configured Policy having target-affinity label for example, using `kubernetes.io/hostname` as a topologyKey in CStorVolumePolicy:
+
+```yaml
+apiVersion: cstor.openebs.io/v1
+kind: CStorVolumePolicy
+metadata:
+  name: csi-volume-policy
+  namespace: openebs
+spec:
+  target:
+    affinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: openebs.io/target-affinity
+            operator: In
+            values:
+            - fio-cstor                              // application-unique-label
+        topologyKey: kubernetes.io/hostname
+        namespaces: ["default"]                      // application namespace
+```
+
+
+Set the label configured in volume policy created above `openebs.io/target-affinity: fio-cstor` on the app pod which will be used to find pods, by label, within the domain defined by topologyKey.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fio-cstor
+  namespace: default
+  labels:
+    name: fio-cstor
+    openebs.io/target-affinity: fio-cstor
+```
+
+
+### Volume Tunable:
+
+Allow users to set available performance tunings in Volume Policy based on their workload. Below are the tunings that can be configured
```

Comment:
How about `Policy allows users to set available performance tunings based on their workload`?

Reply:
done , rephrased the sentence.